examples/heif_convert.cc

```diff
@@ -220,6 +222,9 @@ int main(int argc, char** argv)
           chroma_upsampling = "nearest-neighbor";
         }
         break;
+      case 'v':
+        std::cout << argv[0] << "   libheif version: " << heif_get_version() << std::endl;
```

Comment:
How about only writing the plain version number? That would make it easier for scripts to parse it. And the text in front does not give much information anyway.

Reply:
Or: we could output both, the libheif version and the tool version (which may theoretically be different).

Reply:
You mean like:

```
$ ./heif-convert -v
1.16.1
```

?

Reply:
Yes. Or:
````
$ ./heif-convert -v
libheif: 1.16.1
heif-convert: 1.15.0
````

Not sure if that is useful. Maybe for "broken" installations.

Reply:
As a point of comparison:

```
$ ./avifdec --version
Version: 0.9.3 (dav1d [dec]:0.9.2, libgav1 [dec]:0.17.0, aom [enc/dec]:v3.3.0)
libyuv : available (1807)
```

Reply:
I think I like this:

```
$ ./heif-thumbnailer -v
./heif-thumbnailer: 1.16.1 (libheif: 1.16.1)
$ ./heif-convert -v
./heif-convert: 1.16.1 (libheif: 1.16.1)
```

Reply:
Yes. I think something like this would be most useful. The codec versions are already printed in `--list-encoders` / `--list-decoders`, but other things, like `libsharp-yuv` version or some build configuration could also be useful.

Reply:
One item per line has the advantage that it is easy to grep:
````
heif-convert: 1.15.0
libheif: 1.16.1
libsharp-yuv: N/A
libjpeg: 6.2
...
````

Reply:
OK. This was meant to be an easy fix for a tired Friday night :-)

Will give it further attention over the next few days.

Reply:
Just one additional thought: should we do a plain `-v` / `--version` option for the tool that would be the result of `heif_get_version()` , and a `--dump-configuration` or similar for the build time and run-time versions where we can figure it out?

Reply:
I also was wondering about whether we should have two options. I concluded that I have no clear opinion.
BTW: for consistency, `-v` should then output the tool version and not `heif_get_version()`.